taca/utils/bioinfo_tab.py

```diff
@@ -148,17 +148,21 @@ def get_ss_projects(run_dir):
         logger.error('Cannot find RunParameters.xml or runParameters.xml in the run folder for run {}'.format(run_dir))
         return []
     rp = RunParametersParser(os.path.join(run_dir, run_parameters_file))
-    try:
-        runtype = rp.data['RunParameters']['Setup']['Flowcell']
-    except KeyError:
-        logger.warn('Parsing runParameters to fetch instrument type, '
-                    'not found Flowcell information in it. Using ApplicationName')
+    if 'Setup' in rp.data['RunParameters']:
         try:
-            runtype = rp.data['RunParameters']['Setup'].get('ApplicationName', '')
+            runtype = rp.data['RunParameters']['Setup']['Flowcell']
         except KeyError:
-            logger.warn("Couldn't find 'Setup' or 'ApplicationName' could be Novaseq. Trying 'Application'")
-            runtype = rp.data['RunParameters']['Application']
-
+            logger.warn('Parsing runParameters to fetch instrument type, '
+                        'not found Flowcell information in it. Using ApplicationName')
+            try:
+                runtype = rp.data['RunParameters']['Setup'].get('ApplicationName', '')
+            except KeyError:
+                logger.warn("Couldn't find 'Setup' or 'ApplicationName' could be Novaseq. Trying 'Application'")
```

Comment:
We already check for Setup above in `if 'Setup' in rp.data['RunParameters']:`, don't we? So won't it always have Setup?